src/PathMaster.ts

```diff
@@ -9,7 +9,7 @@ import { Config } from "./Config";
  * If a import url isn't relative
  * and only has ascii + @ in the name it is considered a node module
  */
-const NODE_MODULE = /^([a-z@].*)$/;
+const NODE_MODULE = /^[^:]([a-z@].*)$/;
```

Comment:
> Only matching node module names, but somehow C:\ got in there...

Think it should be `/^([a-z@]*)$/`. There is an extra dot in `.` in the current `/^([a-z@].*)$/` that is wrong :rose: